### PR TITLE
Fix for firstboot enable proxy auth connect

### PR DESCRIPTION
**Pull request by jstavel**
I have added a few new UI elements.
I have changed names of a few UI elements in an application map.
I have changed behavior of test 'firstboot_proxy_auth_connect'

- added a code to handle 'Organization selection' dialog
- added a code to handle irregular error dialog
- commented out a code that verifies that firstboot window exists

```clojure
    (when (tasks/visible? :firstboot-organization-selection)
        (tasks/ui click :firstboot-forward))
      (when (-> :error-dialog (tasks/ui waittillwindowexist 1) bool)
        (tasks/ui click :ok-error)
        (tasks/ui click :firstboot-forward))
      ;;(verify (not (bool (tasks/ui guiexist :firstboot-window))))
      ;;(verify (not (tasks/ui showing? :register-system)))
```

### CODE REVIEW

File: src/rhsm/gui/tasks/tasks.clj [hunk 1]
```diff
@@ -481,7 +481,8 @@
                (bool (ui guiexist :firstboot-window "Subscription Management Registration"))))
   (when server (ui settextvalue :firstboot-server-entry server))
   (if server-default? (ui click :firstboot-server-default))
-  (ui (setchecked (or activation-key activation?)) :firstboot-activation-checkbox)
+  ;; no activation key anymore
+  ;;(ui (setchecked (or activation-key activation?)) :firstboot-activation-checkbox)
```
Comment (rarebreed) [inline src/rhsm/gui/tasks/tasks.clj]: There's no activation key anymore?

File: src/rhsm/gui/tasks/ui.clj [hunk 2]
```diff
@@ -241,21 +242,26 @@ and returns a mapping like :registration-settings -> 'Registration Settings'"
...
+       :firstboot-organization-selection "scpnorgselectionscrolledwindow"
        :firstboot-owner-table "tbl0"})
     (define-elements (windows :firstboot-proxy-dialog)
-      {:firstboot-proxy-checkbox "Proxy Checkbox"
+      {;;:firstboot-proxy-checkbox "Proxy Checkbox"
+       :firstboot-proxy-checkbox "I would like to connect to RedHat Network via an HTTP proxy"
        :firstboot-proxy-location "Proxy Location:"
-       :firstboot-auth-checkbox "Authentication Checkbox"
+       ;;:firstboot-auth-checkbox "Authentication Checkbox"
+       :firstboot-auth-checkbox "Use Authentication with HTTP Proxy"
```
Comment (rarebreed) [inline src/rhsm/gui/tasks/ui.clj]: Same here...is this the locator name for RHEL 7.3 or 6.9?

File: src/rhsm/gui/tasks/tasks.clj (hunk 1 above)
Comment (jstavel) [inline src/rhsm/gui/tasks/tasks.clj]: I have not seen any activation key checkbox here. I can be in some other stage of firstboot now.

File: src/rhsm/gui/tasks/ui.clj [hunk 3]
```diff
@@ -241,21 +242,26 @@ and returns a mapping like :registration-settings -> 'Registration Settings'"
...
        :firstboot-user "account_login"
        :firstboot-pass "account_password"
        :firstboot-autosubscribe "auto_bind"
        :firstboot-system-name "consumer_name"
+       :firstboot-organization-selection "scpnorgselectionscrolledwindow"
        :firstboot-owner-table "tbl0"})
     (define-elements (windows :firstboot-proxy-dialog)
-      {:firstboot-proxy-checkbox "Proxy Checkbox"
+      {;;:firstboot-proxy-checkbox "Proxy Checkbox"
+       :firstboot-proxy-checkbox "I would like to connect to RedHat Network via an HTTP proxy"
```
Comment (jstavel) [inline src/rhsm/gui/tasks/ui.clj]: I have forgetten to check the test against RHEL7. I will do that now.

**Comment by jstavel** [comment]
I have removed old code. Since we do not perform regression tests I think it is not necessary to pay attentition to firstboot in RHEL7.1. I mean it is not necessary to maintain firstboot related names for the old versions of RHEL7.

**Pull request merged**